scripts/production/gcp-create-db

```diff
@@ -56,9 +56,9 @@ if [ ! -v DARK_CLUSTER_CLOUDSQL_PASS ]; then
 fi
 
 gcloud sql instances create "${DARK_CLUSTER_CLOUDSQL_INSTANCE_NAME}" \
-  --database-version=POSTGRES_9_6 \
-  --cpu=8 \
-  --memory=30GB \
+  --database-version=POSTGRES_13 \
+  --cpu=12 \
+  --memory=60GB \
```

Comment:
quick note -- this script isn't actually run unless we're spinning up a _new_ cluster (which we aren't).
Still, I decided to update the pg version referenced, and update cpu/memory to what we currently have in prod.